### PR TITLE
Add Homepage header

### PR DESCRIPTION
**Pull request by Achilles-Dev**

### CODE REVIEW

**Review by Donard97 — CHANGES_REQUESTED** [review]
Hi @Achilles-Dev ,  ✋ 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 💯 

## Invalid Code Review Request

You have submitted a project, that has linter errors. Kindly fix it because linters result should be green. 💪 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,52 @@
...
+- HTML & CSS
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
```
Comment (Deepakdanger) [inline README.md:18]: - [ ] **Required** lets add the code to clone this repo. For example:- 
` git clone https://github.com/Achilles-Dev/hello-microverse.git`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
```
Comment (Deepakdanger) [inline index.html:7]: - [ ] **Required** Let's add the `CSS File link` here so CSS code will reflect in HTML file.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Achilles-Dev, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 
- Good Commits ✔️ 
- No Linter error ✔️ 

## Required Changes ♻️
- [ ] _Check the comments under the review._
- [ ] Let's add some description in the Pull request description.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (Achilles-Dev) [reply index.html:7]: I was not able to add a description to the pull request. Is that okay?

**Review by keroloussamy — APPROVED** [review]
Hi @Achilles-Dev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

clap

## Optional suggestions

- [ ] You can edit your pull request by clicking the Edit button like in the picture 

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**